lib/assesments/assessment_2/fia2milyas/fia2milyas_view/fia2milyas_profile/fia2milyas_custom_widget/fia2milyas_profile_follower.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+
+class FiA2MIlyasProfileFollower extends StatelessWidget {
```

Comment:
Earlier you have used it in a generic way, please change name to something more generic